rllib/BUILD

```diff
@@ -1729,7 +1729,7 @@ py_test(
     name = "examples/batch_norm_model_ddpg_tf",
     main = "examples/batch_norm_model.py",
     tags = ["examples", "examples_B"],
-    size = "small",
+    size = "medium",
```

Comment:
should we revert this?

Reply:
No, it doesn't pass without this. Torch is already at medium and IIRC this was flaky